.changes/v3.9.0/1007-bug-fixes.md

```diff
@@ -0,0 +1,3 @@
+* Add `prefix_length` field to `resourceVcdVappNetwork` to support creating IPv6 vApp networks [GH-1007]
+* Deprecate `netmask` in `resourceVcdVappNetwork` [GH-1007]
```

Comment:
This line should go in a file named `1007-deprecations.md`

Reply:
Updated.

Reply:
Here too: `resourceVcdVappNetwork` - we should state user-facing resource name here.